resources/pdns_authoritative_service_systemd.rb

```diff
@@ -28,10 +28,6 @@
 property :config_dir, String, default: lazy { default_authoritative_config_directory }
 
 action :enable do
-  service 'pdns' do
```

Comment:
I disagree here. If we are to cover the default cause, we should not be forcing a service state upon anyone and leave the control up to the consumer of the resource. If the package installs and starts in some other state, it's your job to configure then use notifications to do this properly. Check out the updated test cookbook for how I achieve this with the postgres backend on the default install. Very simple and it works. Covers the default case and the virtual case.